### PR TITLE
chore(HentaiVN): Change domain.

### PR DESCRIPTION
**Pull request by ndbiaw**
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/H/HentaiVN/metadata.json [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
 	"service": "HentaiVN",
 	"description": {
 		"en": "HentaiVN is the largest Otaku forum and Hentai manga website in Vietnam",
 		"vi_VN": "HentaiVN là website đọc truyện Hentai lớn nhất Việt Nam, nó cũng là diễn đàn dành cho wibu hàng đầu cả nước."
 	},
-	"url": "hentaivn.fun",
-	"version": "2.0.2",
+	"url": {
+		"hentaivn.tv",
+		"hentaivn.site,
```
Comment (DarkVillager) [inline websites/H/HentaiVN/metadata.json:14]:
```suggestion
		"hentaivn.site",
```

File: websites/H/HentaiVN/metadata.json [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
+		"hentaivn.run"
+	},
+	"version": "3.0.0",
```
Comment (DarkVillager) [inline websites/H/HentaiVN/metadata.json:17]:
```suggestion
	"version": "2.1.0",
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Comment by DarkVillager** [comment]
chore(HentaiVN): change domain

File: websites/H/HentaiVN/metadata.json [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
 		"id": "895438058621665331"
 	},
 	"service": "HentaiVN",
 	"description": {
 		"en": "HentaiVN is the largest Otaku forum and Hentai manga website in Vietnam",
 		"vi_VN": "HentaiVN là website đọc truyện Hentai lớn nhất Việt Nam, nó cũng là diễn đàn dành cho wibu hàng đầu cả nước."
 	},
-	"url": "hentaivn.fun",
-	"version": "2.0.2",
+	"url": {
```
Comment (DarkVillager) [inline websites/H/HentaiVN/metadata.json:12]:
```suggestion
	"url": [
```

File: websites/H/HentaiVN/metadata.json [hunk 4]
```diff
@@ -1,16 +1,20 @@
...
+		"hentaivn.tv",
+		"hentaivn.site",
+		"hentaivn.run"
+	},
```
Comment (DarkVillager) [inline websites/H/HentaiVN/metadata.json:16]:
```suggestion
	],
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Review by DarkVillager — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**